spec/assessments/sentenceLengthInTextSpec.js

```diff
@@ -137,6 +152,22 @@ describe( "An assessment for sentence length", function() {
 		expect( assessment.hasMarks() ).toBe( false );
 	} );
 
+	it( "returns the score for all short sentences in Polish", function() {
+		let mockPaper = new Paper( "text", { locale: "pl_PL" } );
+		let sentenceLengthInTextAssessmentPolish = new SentenceLengthInTextAssessment( contentConfiguration( mockPaper.getLocale() ).sentenceLength );
+
+		assessment = sentenceLengthInTextAssessmentPolish.getResult( mockPaper, Factory.buildMockResearcher( [
+			{ sentence: "", sentenceLength: 19 },
+
+		] ), i18n );
+
+		expect( assessment.hasScore() ).toBe( true );
+		expect( assessment.getScore() ).toEqual( 9 );
+		expect( assessment.getText() ).toEqual( "0% of the sentences contain <a href='https://yoa.st/short-sentences' target='_blank'>more than 20 words</a>, " +
+			"which is less than or equal to the recommended maximum of 15%." );
+		expect( assessment.hasMarks() ).toBe( false );
+	} );
+
```

Comment:
Please add specs that actually test for percentage of long sentences, not only for the length of 1 sentence. You need to add a spec that has a text that has < 15% long sentences (GOOD score), between 15 and 20% (OK score) and above 20% (BAD score).